src/corner_kick/README.md

```diff
@@ -0,0 +1,63 @@
+# Corner Kick
+
+Corner Kick is an Electron app that allows for the visualization of UBC Thunderbots' simulator.
+
+## How to use
+
+You can start the visualizer by running:
+
+```
+roslaunch corner_kick corner_kick.launch
+```
+
+## Contribute
+
+### Installing the project
+
+Clone the project and install the dependencies.
+
+```
+rosdep install corner-kick
+```
```

Comment:
I think we should be able to get rid of this entire section? (Dependencies are installed either when `install_dependencies` is run or at build time?)

Reply:
> I think we should be able to get rid of this entire section? (Dependencies are installed either when `install_dependencies` is run or at build time?)

Agreed, this section was removed.